src/Strategy/Python/SetupPy.hs

```diff
@@ -39,3 +43,43 @@ installRequiresParser = do
 
     symbol :: Text -> Parser Text
     symbol = L.symbol space
+
+-- | Parses install requirements listed in setup.cfg
+-- Docs: https://setuptools.pypa.io/en/latest/userguide/declarative_config.html
```

Comment:
Can you add an example of what an install_requires block looks like?

Also, and this is totally optional, but I initially thought we needed to support the comma delimited format and then I figured out that we don't after reading a bit more deeply. It might be worth pointing out that only the dangling form is supported because there can be commas in the version specs.